### ISSUE TITLE
UrBackup not working (Segmentation Fault)

### ISSUE THREAD

**Issue by SonGokuSSJ** [SynoCommunity/spksrc#5607]
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

UrBackup

### Package Version

2.5.27

### Device Model

DS1817

### Device Architecture

ARMv7

### Firmware Version

7.1.1-42962 Update 4

### What happened?

I am writing here, because I was not able to create an issue on [josef109 own Github repository](https://github.com/josef109/spksrc). 

UrBackup crashes. Web Interface does not work. 

The older release [urbackup 2.4.15](https://github.com/josef109/spksrc/releases/tag/2.4.15) installs fine and starts fine. 
The Web Interface is working here (I used the armv7 Install Package). 

### Reproduction steps

```bash
1. Manually install v2.5.27 from josef109: https://github.com/josef109/spksrc/releases
(I tried alpine_7.1, alpine4k_7.1 and arm7_7.1)
2. Klick on "Run" in the Package Center. 
3. Klick on "Open" in the Package Center.
4. Web Gui does not start.
5. Login to NAS via SSH.
6. Trying to start UrBackup via "/volume1/@appstore/urbackup/bin/urbackupsrv run"

Raising maximum file descriptor to 65535 failed. This may cause problems with many clients. (errno=1)
Raising nice-ceiling to 35 failed. (errno=1)
Segmentation fault (core dumped)

7. Trying to start UrBackup via "sudo /volume1/@appstore/urbackup/bin/urbackupsrv run"

Segmentation fault (core dumped)

8. End
```

### Install Log

```
https://pastebin.com/AFVz4uYM
```

**Comment by mreid-tt** [comment]
Hey @SonGokuSSJ, from what I can tell this is not a SynoCommunity supported package. Checking the previous PRs, there did seem to be some work done under #5213 but the effort to launch it as a package never concluded.

Based on what you wrote above, you have an Alpine arch (DS1817) running on DSM 7.1 so the package for armv7-7.0 (under [urbackup 2.4.15](https://github.com/josef109/spksrc/releases/tag/2.4.15)) or armv7-7.1 (under [Urbackup Server 2.5.27](https://github.com/josef109/spksrc/releases/tag/2.5.27)) should be sufficient if the same framework was used to build it. I attempted to install it on my DS916+ running DSM 7.1.1-42962 Update 4 and got an installer log very similar to yours:

[/var/log/packages/urbackup.log](https://pastebin.com/zinYy83N)

The log itself didn't show any obvious errors and seemed to complete successfully with lines like this making this success:

```
2023/02/16 20:13:39	install urbackup 2.5.27-1 End start-stop-status start ret=[0]
```

I also took a look at the service log:

[/var/packages/urbackup/var/urbackup.log](https://pastebin.com/hnyz9m22)

In it I saw some permission errors like:
```
2023-02-16 20:15:26: ERROR: No permission to access "/volume1/@apphome/urbackup/urbackup_tmp_files"
```

As well as failed attempts to adjust some application properties:
```
Starting urbackup command /volume1/@appstore/urbackup/bin/urbackupsrv run -d -v error -u sc-urbackup --pidfile /volume1/@appdata/urbackup/urbackup.pid --logfile /volume1/@appdata/urbackup/urbackup.log
Raising maximum file descriptor to 65535 failed. This may cause problems with many clients. (errno=1)
Raising nice-ceiling to 35 failed. (errno=1)
```

In the end however, I was able to load the web interface. Granted I did use a different installer (x64-7.1) for my Braswell arch. Unfortunately, with this package not being supported by the SynoCommunity and me being unable to recreate the issue there is not much more I can suggest. It may be possible that somewhere between version 2.4.15 and 2.5.27, the original developer dropped support for the ARMv7 arch but the folks at [UrBackup](https://www.urbackup.org/) would need to weigh in on that.

Tagging @josef109 so that if he sees this he can maybe lend some additional insight.

**Comment by SonGokuSSJ** [comment]
@mreid-tt 
Thank you for the reply. I know, that it is not supported / created here, but I saw the Pull Request by @hgy59, so I thought I could post it here, because I did not see an option to create an issue on josef109 repository. I did open the new issue, because I thought I should not post on a pull request either. 

The 2 Raising errors seem to go away if launching via sudo. but the segmentation fault stays. Could well be a permission issue. I also thought of trying to upgrade via the newer version package, but the Segmentation fault stopped me from doing it, because it looks to me, that it is a problem with the version and not a permission issue. 

Anyway thank you for your input and "poking" josef109. Maybe he sees this post in some days (though from the look of hgy59's pull request it seems josef109 has been absent a little-bit longer.
